packages/blockchain-wallet-v4-frontend/src/components/Form/PhoneNumberBox/index.js

```diff
@@ -110,7 +109,7 @@ class PhoneNumberBox extends React.Component {
 }
 
 PhoneNumberBox.propTypes = {
-  countryCode: PropTypes.object.isRequired
+  countryCode: PropTypes.string
```

Comment:
Wanted to focus on the modals and then if necessary go through the components folder - always trying to balance the "make this repo better" with "this will be replaced not so far in the future"